circleci: failed tests (14027): TestSetupRangeTree

The following test appears to have failed:

[#14027](https://circleci.com/gh/cockroachdb/cockroach/14027):

```
I160308 03:57:23.337646 client/db.go:491  failed batch: node unavailable; try another peer
W160308 03:57:23.337744 kv/dist_sender.go:627  failed to invoke AdminSplit [/Table/13/0,/Min): storage/replica_command.go:1385: split at key /Table/13 failed: node unavailable; try another peer
I160308 03:57:23.337797 client/db.go:491  failed batch: storage/replica_command.go:1385: split at key /Table/13 failed: node unavailable; try another peer
E160308 03:57:23.337870 storage/queue.go:440  [split] range=1 [/Min-/Table/11): error: storage/split_queue.go:100: unable to split range=1 [/Min-/Table/11) at key "/Table/13/0": storage/replica_command.go:1385: split at key /Table/13 failed: node unavailable; try another peer
I160308 03:57:23.337949 storage/engine/rocksdb.go:141  closing in-memory rocksdb instance
--- FAIL: TestSetupRangeTree (0.04s)
	client_range_tree_test.go:188: tree roots do not match - expected:root_key:""  actual:root_key:"\223" 
=== RUN   TestTree
W160308 03:57:23.345756 gossip/gossip.go:921  not connected to cluster; use --join to specify a connected node
W160308 03:57:23.346214 gossip/gossip.go:394  bad address {localhost 1}: gossip/resolver/resolver.go:95: unknown address network "localhost" for 1
I160308 03:57:23.347335 raft/raft.go:459  [group 1] 1 became follower at term 5
I160308 03:57:23.347397 raft/raft.go:235  [group 1] newRaft 1 [peers: [1], term: 5, commit: 10, applied: 10, lastindex: 10, lastterm: 5]
I160308 03:57:23.347424 raft/raft.go:539  [group 1] 1 is starting a new election at term 5
I160308 03:57:23.347451 raft/raft.go:472  [group 1] 1 became candidate at term 6
I160308 03:57:23.347475 raft/raft.go:521  [group 1] 1 received vote from 1 at term 6
I160308 03:57:23.347505 raft/raft.go:500  [group 1] 1 became leader at term 6
--
I160308 03:57:40.290276 stopper.go:273  draining; tasks left:
2      gossip/infostore.go:285
I160308 03:57:40.290311 stopper.go:273  draining; tasks left:
1      gossip/infostore.go:285
--- PASS: Example_rebalancing (0.16s)
FAIL
FAIL	github.com/cockroachdb/cockroach/storage	33.877s
=== RUN   TestBatchBasics
I160308 03:57:24.173164 storage/engine/rocksdb.go:141  closing in-memory rocksdb instance
--- PASS: TestBatchBasics (0.00s)
=== RUN   TestBatchGet
I160308 03:57:24.185202 storage/engine/rocksdb.go:141  closing in-memory rocksdb instance
--- PASS: TestBatchGet (0.01s)
=== RUN   TestBatchMerge
I160308 03:57:24.189345 storage/engine/rocksdb.go:141  closing in-memory rocksdb instance
--- PASS: TestBatchMerge (0.00s)
=== RUN   TestBatchProto

```
Please assign, take a look and update the issue accordingly.

--- a/storage/client_range_tree_test.go
+++ b/storage/client_range_tree_test.go
@@ -176,7 +176,7 @@ func verifyProperty5(t *testing.T, nodes map[string]roachpb.RangeTreeNode, testN
 // tree and first node. SetupRangeTree is called via store.BootstrapRange.
 func TestSetupRangeTree(t *testing.T) {
 	defer leaktest.AfterTest(t)()
-	t.Skip("TODO(bram): #4968")
+	defer config.TestingDisableTableSplits()()
 	store, stopper := createTestStore(t)
 	defer stopper.Stop()
 	db := store.DB()

